app/views/integrations/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :title, "Integrations" %>
+
+<% content_for :content do %>
+  <div class="row">
+    <div class="col-xs-12">
+      <h2>NetSuite</h2>
+
+      <div class="row">
+        <div class="col-xs-12">
+          <h4>Total Inventory Value</h4>
+          <p><%= number_to_currency(@total_inventory_value, precision: 2) %></p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-xs-12">
+          <h4>Exports</h4>
+
+          <%= link_to "NetSuite Donations Export",
+            net_suite_donation_export_reports_path,
+            { class: "btn btn-default", data: { no_turbolink: true } } %>

```

Comment:
The last arguments of a method doesn't need the hash braces since it is automatic (I believe rubocop would complain about this in non-erb files, so I think we should honor the same styles in the erb files).

The alignment seems a little off to me.... maybe align the arguments with eachother, otherwise use 1 line?

So perhaps:
```
          <%= link_to "NetSuite Donations Export",
                      net_suite_donation_export_reports_path,
                      class: "btn btn-default", data: { no_turbolink: true } %>
```

The same comment applies below.